sites-enabled/10-www.freecodecamp.org.conf

```diff
@@ -684,340 +476,10 @@ server {
-  }
-  # location /haitian/ {
-  #   # the trailing hash is needed here.
-  #   proxy_pass http://client-hat/;
-  #   include snippets/common/proxy-params.conf;
-  # }
-
-  # This request is causing server errors within ghost, so we reject here to
-  # reduce load. The ^~ prevents regex locations from matching these urls.
-  location ^~ /haitian/news/ghost/api/v3/admin/snippets/ {
-    return 500;
-  }
-
-  # reverse proxy news (haitian) - Ghost
-  location ~ ^/haitian/news/(ghost|content|p)(/.*)?$ {
-    if (-f /etc/nginx/maintenance/NEWS-HAT.txt) {
-      return 503;
-    }
-    access_log /var/log/nginx/access.log combined if=$log_4xx_5xx;
-    proxy_pass http://news-hat;
-    include snippets/common/proxy-params.conf;
-  }
-
-  # reverse proxy news (haitian) - JAMStack
-  location /haitian/news {
-    if (-f /etc/nginx/maintenance/NEWS-HAT.txt) {
-      return 503;
-    }
-    access_log /var/log/nginx/access.log combined if=$log_4xx_5xx;
-
-    proxy_cache            NEWS_CACHE_PRD_HAT;
-    proxy_cache_valid      200  1h;
-    proxy_cache_use_stale  error timeout invalid_header updating
-                           http_500 http_502 http_503 http_504;
-
-    proxy_pass http://news-hat;
-    include snippets/common/proxy-params.conf;
-
-    # proxy_pass https://haitian.news.cdn.freecodecamp.org/;
-    # proxy_set_header X-Forwarded-Host   haitian.news.cdn.freecodecamp.org;
-    # proxy_set_header Host   haitian.news.cdn.freecodecamp.org;
-
-    # include snippets/common/proxy-params.azure.conf;
-  }
-
-    # reverse proxy client (german)
-  location /german/ {
-    # the trailing hash is needed here.
-    proxy_pass http://client-ger/;
-    include snippets/common/proxy-params.conf;
-  }
-
-  # This request is causing server errors within ghost, so we reject here to
-  # reduce load. The ^~ prevents regex locations from matching these urls.
-  location ^~ /german/news/ghost/api/v3/admin/snippets/ {
-    return 500;
-  }
-
-  # reverse proxy news (german) - Ghost
-  location ~ ^/german/news/(ghost|content|p)(/.*)?$ {
-    if (-f /etc/nginx/maintenance/NEWS-GER.txt) {
-      return 503;
-    }
-    access_log /var/log/nginx/access.log combined if=$log_4xx_5xx;
-    proxy_pass http://news-ger;
-    include snippets/common/proxy-params.conf;
-  }
-
-  # reverse proxy news (german) - JAMStack
-  location /german/news {
-    if (-f /etc/nginx/maintenance/NEWS-GER.txt) {
-      return 503;
-    }
-    access_log /var/log/nginx/access.log combined if=$log_4xx_5xx;
-
-    proxy_cache            NEWS_CACHE_PRD_GER;
-    proxy_cache_valid      200  1h;
-    proxy_cache_use_stale  error timeout invalid_header updating
-                           http_500 http_502 http_503 http_504;
-
-    proxy_pass https://german.news.cdn.freecodecamp.org/;
-    proxy_set_header X-Forwarded-Host   german.news.cdn.freecodecamp.org;
-    proxy_set_header Host   german.news.cdn.freecodecamp.org;
-
-    include snippets/common/proxy-params.azure.conf;
-  }
```

Comment:
German is a special and problematic deployment for now because we have learn sites in german, but the news-instance is retired. 

So ideally, we will have this back soon, but it makes the wiring of it very tedious during the initial cutover.

Reply:
I'll bring it up (hopefully today-ish).